resources/types/Pangenome.yaml

```diff
@@ -0,0 +1,21 @@
+global-object-type: Pangenome
+storage-type: WS
+storage-object-type: KBaseGenomes.Pangenome
+versions:
+- indexing-rules:
+  - path: name
+    full-text: true
+  - path: type
+    keyword-type: string
+    ui-name: Pangenome Type
+  - path: genome_refs/{size}
+    keyword-type: integer
+    key-name: genomes
+  - path: orthologs/{size}
+    keyword-type: integer
+    key-name: orthologs
+  # I expect this won't work but I'm including to spur discussion on what might
```

Comment:
So does this work?

Reply:
This does not work in part because of the "key-name" issue. I'm going to scratch it for this version of the specs